sandbox/registration-processor-mz.properties

```diff
@@ -548,9 +557,12 @@ mosip.regproc.cmd-validator.server.port=8089
 mosip.regproc.cmd-validator.eventbus.port=5716
 mosip.regproc.cmd-validator.server.servlet.path=/registrationprocessor/v1/cmdvalidator
 
-mosip.registration.processor.validate-machine=true
-mosip.registration.processor.validate-device=true
-mosip.registration.processor.validate-center=true
+#To enable center validation for the mentioned processes/registration types
+mosip.regproc.cmd-validator.center-validation.processes=NEW,UPDATE,LOST,BIOMETRIC_CORRECTION
+#To enable machine validation for the mentioned processes/registration types
+mosip.regproc.cmd-validator.machine-validation.processes=NEW,UPDATE,LOST,BIOMETRIC_CORRECTION
+#To enable device validation for the mentioned processes/registration types
+mosip.regproc.cmd-validator.device-validation.processes=NEW,UPDATE,LOST,BIOMETRIC_CORRECTION
```

Comment:
1. Leave 1 space between # and first letter of the comment
2. Change the comment as this format"Processes to enable center validation, for processes not mentioned here center validation will be skipped"

Reply:
updated

Reply:
resolved